packages/edit-site/src/components/sidebar-navigation-screen-template/index.js

```diff
@@ -29,6 +29,7 @@ export default function SidebarNavigationScreenTemplate() {
 			actions={
 				<Button
 					variant="primary"
+					className="edit-site-sidebar-navigation-screen__button"
```

Comment:
```suggestion
					className="edit-site-sidebar-navigation-screen__edit"
```

Let's rename to match a bit more the other classes we have in sidebar(ex: `edit-site-sidebar-navigation-screen__back`).